src/ContentCreator.php

```diff
@@ -751,8 +751,11 @@ protected function getSampleData($type, $random = TRUE) {
 
   /**
    * Import generated CSV files into database.
+   *
+   * @param bool $gracefully_for_disk_storage
+   *   Flag to free disk space when it's possible.
    */
-  public function importCsvFiles() {
+  public function importCsvFiles($gracefully_for_disk_storage = TRUE) {
```

Comment:
I do not understand the variable name. Why not use `$keep_content_files` from the command line and invert the logic